test/test_status.c

```diff
@@ -0,0 +1,70 @@
+#include "status.h"
+
+#include <stdbool.h>
+#include <stdlib.h>
+
+#include "misc.h"
+#include "unity.h"
+
+void setup_test(void) {}
```

Comment:
I feel like in order to be consistent with ``struct`` and array initialization, we should have a space in between

```c
void setup_test(void) { }
```

If it's not in our style guide, we should fix that.